### PR TITLE
update read me for level 1 coding challenge

### PR DESCRIPTION
**Pull request by ruddy-m**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by TheAbstract — CHANGES_REQUESTED** [review]
@ruddy-m I think it might be better to either have consistent task function names (`task1`, `task2`, ..., `task7`) or have specific names for each task like (`combine`, `square`, etc.). 
So for example either one of these  would be fine but not a mixture of the two 
```
task1    multiples
task2    three
task3    sixtyfive
task4    square
task5    triangle
task6    longest
task7    combine
```

File: content/projects/katas/level-1/_index.md [hunk 1]
```diff
@@ -75,25 +75,25 @@ If you follow the simple instructions in the "what does success look like?" sect
...
+Write a function `task3` that takes 2 numbers as input.
 If either of the numbers is 65, or if the sum of the numbers is 65 then return true. Otherwise, return false.
 
-## Task 1.4
+## Task 4
 
-Write a function, name it `square`. It takes in an integer and then prints out a square using the hash character.
+Write a function `task4`, name it `square`. It takes in an integer and then prints out a square using the hash character.
 
 For example, `square(2)` should print:
```
Comment (sheenarbw) [inline content/projects/katas/level-1/_index.md:100]: Write a function `task4`, name it `square`. I'm not sure if the function is meant to be named task4 or square

File: content/projects/katas/level-1/_index.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ For example, `square(4)` should print:
 ####
 ```
 
-## Task 1.5
+## Task 5
 
 For example, `triangle(2)` should print:
```
Comment (sheenarbw) [inline content/projects/katas/level-1/_index.md]: should this say `task5`? or `triangle`? I see in the automaker config you named it `triangle`, I'm just concerned that it's not consistent with all the tasks before.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
There's a little polish needed here and there. And it would be good to add something to tell the learner about the expected directory structure and file names

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**